src/js/jsx/Scrim.jsx

```diff
@@ -249,21 +287,28 @@ define(function (require, exports, module) {
             var document = this.state.document,
                 disabled = document && document.unsupported,
                 transform = this.state.transform,
+                isDragging = this.state.isDragging,
                 overlays = !disabled && this.state.overlaysEnabled,
                 transformString = this._getTransformString(transform),
-                toolOverlay = (overlays && transform) ? this._renderToolOverlay(transformString) : null,
+                toolOverlay = (overlays && transform && !isDragging) ? this._renderToolOverlay(transformString) : null,
```

Comment:
This check to whether render tool overlay is broken across two places. Can you consolidate them in renderToolOverlay function instead?